doc/api/deprecations.md

```diff
@@ -2361,6 +2361,15 @@ changes:
     description: Runtime deprecation.
 -->
 
+<a id="DEP0126"></a>
```

Comment:
1. This seems to break other deprecation section without adding its own description.
2. We usually assign a placeholder `DEP0XXX` till landing.
3. PR URL should be https://github.com/nodejs/node/pull/26717
4. I am not sure this is common to start with runtime deprecation.